nullaway/src/main/java/com/uber/nullaway/dataflow/CoreNullnessStoreInitializer.java

```diff
@@ -30,7 +30,7 @@ public NullnessStore getInitialStore(
       Context context,
       Types types,
       Config config) {
-    if (parameters == null) {
+    if (underlyingAST.getKind().equals(UnderlyingAST.Kind.ARBITRARY_CODE)) {
```

Comment:
Is this a check for an initializer block specifically? What happens to fields initialize with `T x = [expression]`? Wasn't that also a case we were covering by looking for `parameters == null`?

(I am probably wrong here, since I haven't thought about this code in a while, but want to make sure I understand the change)

Reply:
Here are the docs for the `ARBITRARY_CODE` case:

https://github.com/typetools/checker-framework/blob/de2f18948fb5ef6a4cf47163c8a6457404bc2cd7/dataflow/src/main/java/org/checkerframework/dataflow/cfg/UnderlyingAST.java#L24-L25

I _believe_ this case covers both initializer blocks and expressions.  @mernst do you know?  In any case all our tests from before still pass, and the check for `parameters == null` no longer works.

Reply:
Yes, `ARBITRARY_CODE` is for initializer blocks.  The right-hand-sides of initializers will have an expression type; more relevantly, they don't have their own `UnderlyingAST`.